src/httpserver/http_request.hpp

```diff
@@ -33,19 +36,9 @@
 
 #include "httpserver/http_utils.hpp"
 
-struct MHD_Connection;
```

Comment:
Similar to the comment above. I am not sure that I prefer a full-fledged include over a forward-declaration